### PR TITLE
skillcalculator: Add New Amethyst Items

### PR DESCRIPTION
**Pull request by Vetricci**
Add amethyst darts & dart tips to fletching and crafting calculators respectively.

### CODE REVIEW

File: runelite-client/src/main/resources/net/runelite/client/plugins/skillcalculator/skill_crafting.json [hunk 1]
```diff
@@ -757,6 +757,12 @@
       "name": "Light Orb",
       "xp": 70
     },
+    {
+      "level": 89,
+      "icon": 25853,
+      "name": "Amethyst Dart Tips",
```
Comment (Nightfirecat) [inline runelite-client/src/main/resources/net/runelite/client/plugins/skillcalculator/skill_crafting.json:763]:
```suggestion
      "name": "Amethyst Dart Tip",
```

File: runelite-client/src/main/resources/net/runelite/client/plugins/skillcalculator/skill_crafting.json [hunk 2]
```diff
@@ -757,6 +757,12 @@
...
+      "xp": 60
```
Comment (Nightfirecat) [inline runelite-client/src/main/resources/net/runelite/client/plugins/skillcalculator/skill_crafting.json:764]: I'm not sure that this is the right value to use. Presently, we use the xp given from a single of the item created, even if it's not possible to only create one of the item--case in point, arrow shafts. That would make this value 7.5, not 60.

**Review by Nightfirecat — CHANGES_REQUESTED** [review]

File: runelite-client/src/main/resources/net/runelite/client/plugins/skillcalculator/skill_crafting.json (hunk 2 above)
Comment (Vetricci) [reply runelite-client/src/main/resources/net/runelite/client/plugins/skillcalculator/skill_crafting.json:764]: The value is consistent with the existing amethyst items (arrowtips, javelin heads, bolt tips). For crafting you're looking for the number of amethyst to process, each providing 60 xp.

Comment (Nightfirecat) [reply runelite-client/src/main/resources/net/runelite/client/plugins/skillcalculator/skill_crafting.json:764]: Each of the items you mentioned can be processed individually, unlike the arrow shaft case I mentioned. You cannot craft fewer than 8 amethyst dart tips.

**Pull request merged**